app/models/account.rb

```diff
@@ -1,3 +1,9 @@
 class Account < ApplicationRecord
   belongs_to :project
+  belongs_to :account_manager, class_name: 'User'
+
+  validates :name, presence: true
+  validates :services, presence: true, allow_nil: false
+  validates :account_manager_id, allow_nil: false
```

Comment:
Rails 6 óta, ha `belongs_to`-val van definiálva egy modell kapcsolat, akkor automatikusan validáljuk annak az ID mezőnek a jelenlétét. Szóval erre a fenti sorra nincs szükség.

(Nice to know, `belongs_to :something, optional: true` <= így tudod elérni, hogy ne legyen kötelező kitölteni a `something_id` mezőt.)